### PR TITLE
update 15five region

### PR DESCRIPTION
**Pull request by caleb15**
we are hiring people in central / south america now too.

This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [x] I am an employee of the company mentioned and confirm all included details are correct
- [x] This PR contains housekeeping only (URL edits, copy changes etc)
- [x] You know your alphabet - company is listed in alphabetical order in the README
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Comment by caleb15** [comment]
next PR updates readme.

File: company-profiles/15five.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Over half of company works remote. We communicate via [slack](https://www.slack.
 
 ## Region
 
-North America and Europe. 
+The Americas and Europe.
```
Comment (c0b) [inline company-profiles/15five.md]: are you using `The Americas` to mean including `South America` as well? but from where do you see ? Could you give a link of information source?
I've just checked their open list, I only see the US based remote in `The Americas` region, not even including Canada,

- https://www.15five.com/careers/
- https://jobs.lever.co/15five

**Comment by c0b** [comment]
and do you want to merge with #573 ? the changes to one single company is better in a single PR?

**Comment by caleb15** [comment]
Yes, including South America.  I'm a employee of 15Five, I just asked the
CTO about the hiring policy.  The webpage should be updated soon.

**Comment by caleb15** [comment]
572 and 573 should be accepted at the same time, correct.

**Comment by dougaitken** [comment]
@caleb15 can you update this PR with the change from #573 please?

**Comment by caleb15** [comment]
@dougaitken done.

**Pull request merged**